### ISSUE TITLE
GLTron

### ISSUE THREAD

**Issue by Pythonic456** [Botspot/pi-apps#307]
You can `sudo apt install gltron` to install GLTron. I think this would be a good thing
to add to Pi-Apps because it's not obvious how to play Tron on a Pi.

**Comment by Itai-Nelken** [comment]
I'm pretty sure Botspot won't approve it, see #185.
<br>
<img width="624" alt="Screen Shot 2021-02-01 at 1 18 37 PM" src="https://user-images.githubusercontent.com/70802936/106452070-0fea1780-6490-11eb-957c-3f37117fdf3f.png">

**Comment by Pythonic456** [comment]
Hmm, I'm not sure what @BotSpot will say, but some people just don't know how to use apt. Just thought it might be a nice addition.

**Comment by Itai-Nelken** [comment]
> Hmm, I'm not sure what @Botspot will say, but some people just don't know how to use apt. Just thought it might be a nice addition.

I thought more or less the same about SimpleScreenRecorder and it wasn't approved (#27).

**Comment by Botspot** [comment]
> some people just don't know how to use apt. Just thought it might be a nice addition.

Agreed. APT is not good for finding you a new app you didn't know existed before.

When Pi-Apps started, I didn't want packages entering the app store because they would **clutter** the app list. But now that we have _categories_, that reason is negligible.  
So now my reason for not including packages is because **there are so many of them** and because adding **each one takes too long**.
